### PR TITLE
STEP6: upload the docker image to the registry

### PR DESCRIPTION
**Pull request by noliver-ykc**
## What
uncommented the command to push to docker.
// Accidentally pushed to mercari main so i am now resending this request

## CHECKS :warning:

Please make sure you are aware of the following.

- [x] **The changes in this PR doesn't have private information

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -37,10 +37,10 @@ jobs:
...
-#      with:
-#       context: <go or python>
-#       push: true
-#       tags: ${{ steps.meta.outputs.tags }}
-#       labels: ${{ steps.meta.outputs.labels }}
\ No newline at end of file
+   - name: Build and push Docker image
+     uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+     with:
+      context: <go or python>
```
Comment (karuna) [inline .github/workflows/build.yml]: which one?

Comment (noliver-ykc) [reply .github/workflows/build.yml]: ahh i forgot to change that, ill edit it soon!

**Comment by iggyray** [comment]
`build.yml` lgtm, but did you enable github actions in your repo? Can't seem to see the workflow being executed in the `checks` section (between `commits` and `files changed`)

**Comment by noliver-ykc** [comment]
> `build.yml` lgtm, but did you enable github actions in your repo? Can't seem to see the workflow being executed in the `checks` section (between `commits` and `files changed`)
Hi! It shows my actions displaying on github but it also looks like theyve all failed so im a bit at a loss. 

キャプチャ

**Comment by iggyray** [comment]
Screenshot 2023-05-18 at 11 40 02 AM
ahh! `.yml` files are sensitive to indenting, and it seems like your `Build and push docker image` step is misaligned

**Review by iggyray — APPROVED** [review]
lgtm!

**Review by karuna — APPROVED** [review]

**Comment by noliver-ykc** [comment]
> Screenshot 2023-05-18 at 11 40 02 AM ahh! `.yml` files are sensitive to indenting, and it seems like your `Build and push docker image` step is misaligned

that fixed it!! thanks so mch. i swear i was gonna loose my mind

**Pull request merged**